lib/listcoef.gd

```diff
@@ -101,26 +101,53 @@ DeclareOperation(
 #O  MultRowVector( <list>, <mul> )
 ##
 ##  <#GAPDoc Label="MultRowVector">
+##  <#GAPDoc Label="MultRowVectorLeft">
+##  <#GAPDoc Label="MultRowVectorRight">
 ##  <ManSection>
 ##  <Oper Name="MultRowVector" Arg='list1, [poss1, list2, poss2, ]mul'/>
+##  <Oper Name="MultRowVectorLeft" Arg='list1, [poss1, list2, poss2, ]mul'/>
+##  <Oper Name="MultRowVectorRight" Arg='list1, [poss1, list2, poss2, ]mul'/>
+##  <Returns>nothing</Returns>
 ##
 ##  <Description>
-##  The five argument version of this operation replaces
+##  Note that <C>MultRowVector</C> is only a Synonym for
+##  <C>MultRowVectorLeft</C>.
+##  </P>
+##  These operations multiply <A>list1</A> with <A>mul</A> in-place and write
+##  the result to <A>list1</A>.
```

Comment:
"... [modify] list1 ... in-place and write the result to list1" seems a bit redundant... I'd stop after "in-place". Or, if you really want to elaborate on that, perhaps: "... in-place, i.e., the result overwrites list1" or so -- here, the "i.e." makes clear the following is just a clarification of what came before.